app/models/card.rb

```diff
@@ -1,4 +1,13 @@
 # Model for keeping card data.
-
 class Card < ApplicationRecord
+  before_validation { self.review_date = Time.now + 3.days }
```

Comment:
before create  должно быть, иначе дата будет обновляться даже если карточка просто обновлена

Reply:
big thanks. corrected: before_validation(on: :create) { self.review_date = Time.now + 3.days }
before_create - doesn't work because of callbacks flow: 
... (1) before_validation; (-) validate; (2) after_validation; (3) before_save; (4) before_create; 
(-) create ...
http://api.rubyonrails.org/classes/ActiveRecord/Callbacks.html